Sources/Views/PricingLanding.swift

```diff
@@ -26,17 +26,37 @@ public func pricingLanding(
     + plansAndPricing(
       allEpisodeCount: allEpisodeCount,
       episodeHourCount: episodeHourCount,
-      freeEpisodeCount: freeEpisodeCount
+      freeEpisodeCount: freeEpisodeCount,
+      subscriberState: subscriberState
     )
     + whatToExpect
     + faq
     + whatPeopleAreSaying
     + featuredTeams
-    + footer(currentUser: currentUser)
+    + footer(allEpisodeCount: allEpisodeCount, currentUser: currentUser, subscriberState: subscriberState)
 }
 
 func ctaColumn(currentUser: User?, subscriberState: SubscriberState) -> [Node] {
-  guard !subscriberState.isActive else { return [] }
+  guard currentUser == nil || subscriberState.isActive else { return [] }
+
+  let title = subscriberState.isActive
+    ? "You‘re already a subscriber!"
+    : "Start with a free episode"
+
+  let ctaButton = subscriberState.isActive
+    ? a(
+      [
+        href(path(to: .account(.index))),
+        `class`([Class.pf.components.button(color: .white)])
+      ],
+      ["Manage your account"]
```

Comment:
this creates a CTA for active subscribers you visit the pricing landing page:

<img width="953" alt="image" src="https://user-images.githubusercontent.com/135203/63179572-6d7b7180-c01a-11e9-834e-934ee074e4a6.png">